sql/CMakeLists.txt

```diff
@@ -106,7 +114,10 @@ function(cat_files SRC_FILE_LIST OUTPUT_FILE)
 endfunction()
 
 # Generate the extension file used with CREATE EXTENSION
-cat_files("${CATALOG_SOURCE_FILES_VERSIONED};${SOURCE_FILES_VERSIONED}" ${CMAKE_CURRENT_BINARY_DIR}/${INSTALL_FILE})
+cat_files(
+  "${CATALOG_SOURCE_FILES_VERSIONED};${SOURCE_FILES_VERSIONED};${IMMUTABLE_API_SOURCE_FILES_VERSIONED}"
```

Comment:
Is there a weird extra return right here?

Reply:
The line was getting too long that is why I split this to multiple lines.
My thought was that if we later add more parts to the build pipeline, we can concat the string form the next line